gradio/themes/base.py

```diff
@@ -224,65 +224,65 @@ def expand_shortcut(shortcut, mode="color", prefix=None):
     def set(
         self,
         *,
+        # Body
```

Comment:
BTW I think the refactoring broke `blocks_kitchen_sink` and my dracula theme. Get the same error in both.
```bash
❯ python demo/blocks_kitchen_sink/run.py
Traceback (most recent call last):
  File "demo/blocks_kitchen_sink/run.py", line 7, in <module>
    base_theme = gr.themes.Base()
  File "/Users/freddy/sources/gradio/gradio/themes/base.py", line 222, in __init__
    self.set()
  File "/Users/freddy/sources/gradio/gradio/themes/base.py", line 714, in set
    self.background_accent = background_accent or getattr(
NameError: name 'background_accent' is not defined
```

Reply:
whoops not sure when that broke, fixed now. renamed background_accent to color_accent since it was being used for text color as well. you may need to fix dracula if you changed that var.